src/zenml/integrations/kubeflow/orchestrators/kubeflow_orchestrator.py

```diff
@@ -291,7 +315,7 @@ def list_manual_setup_steps(
         """Logs manual steps needed to setup the Kubeflow local orchestrator."""
         global_config_dir_path = zenml.io.utils.get_global_config_directory()
         kubeflow_commands = [
-            f"> k3d cluster create CLUSTER_NAME --registry-create {container_registry_name} --registry-config {container_registry_path} --volume {global_config_dir_path}:{global_config_dir_path}\n",
+            f"> k3d cluster create CLUSTER_NAME --image {local_deployment_utils.K3S_IMAGE_NAME} --registry-create {container_registry_name} --registry-config {container_registry_path} --volume {global_config_dir_path}:{global_config_dir_path}\n",
```

Comment:
Just out of curiosity: why did you need to specify the image here explicitly ?

Reply:
Every k3d release comes bundled with a default k3s image (which in turn refers to a kubernetes version). Users with a newer k3d installation had problems as the default image had a kubernetes version >= 1.22.0 and KFP is only compatible with < 1.22.0